### PR TITLE
fix(pci): display activate region warning message

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #... MANAGER-10314
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ (not applicable)

## Description
Display activate region warning message in gateway creation. 

:flags: Translations:
- [x] TRANS-48055

### CODE REVIEW

**Review by stif59100 — DISMISSED** [review]

**Review by stif59100 — DISMISSED** [review]

**Review by vks-ovh — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/gateways/add/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -36,5 +36,6 @@
...
   "pci_projects_project_public_gateways_add_success": "Une nouvelle Gateway a été créée avec succès.",
   "pci_projects_project_public_gateways_add_error": "Une erreur est survenue lors de la création de la Gateway: {{ message }}",
-  "pci_projects_project_public_gateways_add_creating_wait_message": "Votre Gateway est en cours de création. La configuration de tous les composants peut prendre quelques minutes. Veuillez ne pas quitter ou rafraîchir cette page car cela pourrait avoir un impact sur le processus de création."
+  "pci_projects_project_public_gateways_add_creating_wait_message": "Votre Gateway est en cours de création. La configuration de tous les composants peut prendre quelques minutes. Veuillez ne pas quitter ou rafraîchir cette page car cela pourrait avoir un impact sur le processus de création.",
+  "pci_projects_project_public_gateways_add_region_activate": "Cette région n'est pas encore activée. Vous pouvez l'activer sur l'onglet <a href='{{regionsLink}}'>Quota and localisation</a> puis revenir créer votre Gateway."
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/gateways/add/translations/Messages_fr_FR.json:40]: **nitpick:** Avoid HTML in translation files

Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/project/gateways/add/translations/Messages_fr_FR.json]: Done Anoop

File: packages/manager/modules/pci/src/projects/project/gateways/add/add.html [hunk 2]
```diff
@@ -97,6 +97,26 @@
...
+                rel="noopener"
+            >
+                <span
+                    data-translate="pci_projects_project_public_gateways_add_region_activate_line2"
+                ></span>
+            </a>
+            <span
+                data-translate="pci_projects_project_public_gateways_add_region_activate_line3"
+            ></span>
+        </div>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/gateways/add/add.html:119]: **niptick:** Use `oui-message` instead of `div` with classes.

Comment (sachinrameshn) [reply packages/manager/modules/pci/src/projects/project/gateways/add/add.html]: Done

**Review by mohammed-zahaf — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**